Ninano/Ninano/CalendarDetail/CalendarDetailViewController.swift

```diff
@@ -0,0 +1,158 @@
+//
+//  CalendarDetailViewController.swift
+//  Ninano
+//
+//  Created by Eunbee Kang on 2022/07/18.
+//
+
+import UIKit
+
+class CalendarDetailViewController: UIViewController {
```

Comment:
```suggestion
final class CalendarDetailViewController: UIViewController {
```

더이상 상속되지 않는다는 것을 명확히 명시하기위해 final을 붙이는 것이 좋아보여요 ~

Reply:
반영하였습니다! 감사합니다!!